test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -4026,6 +4026,19 @@ def forward(self, x, y, z, ind):
         ind = torch.tensor(-2, dtype=torch.long)
         self.run_test(GetItemModel(), (x, y, z, ind))
 
+    def test_item(self):
+        class M(torch.nn.Module):
+            def comp(self, a: int, b: int):
```

Comment:
What is the use of 'comp' ?

Reply:
good catch, test code while debugging, removed.